A collection of minor updates for 2022

mwcraig opened the pull request:
This also contains a couple of fixes for ipywidgets 8. Closes #177

marimeireles commented:
On it! :)

marimeireles commented:
Hey @mwcraig is not working at all to me.
I had a problem with `couldn't find version 0.2.19 of voila` so I changed to the latest `voila==0.3.5` and had a lot of conflicts. Changed back to `0.2.10` and looks fine. We surely need the 2.19?

ianhi commented:
The readme claims that: 

> The installation instructions were tested on an up-to-date version of Windows 10 Professional. If you encounter any issues on Windows please open an issue or contact us through slack.

Is that actually true? Does anyone have windows on which to test this?

ianhi commented on environment.yml:
```diff
@@ -1,30 +1,32 @@
   - scipy
   - sidecar=0.5.1
-  - voila=0.2.10
+  - voila=0.2.19
```
The last in the `0.2.x` series is 16

```suggestion
  - voila=0.2.16
```

mwcraig replied:
Thanks -- ended up changing it to `0.3.5` and dropping `voila-vuetify` and `voila-material`

ianhi commented on environment.yml:
```diff
@@ -1,30 +1,32 @@
-  - ipyleaflet=0.14.0
-  - ipympl=0.7.0
+  - ipyleaflet=0.16.0
+  - ipympl=0.9.1
```
I will need to make a new release of ipympl if we switch to ipywidgets 8. The current release has a gate on `ipywidgets<8`.
I started work on that here: https://github.com/matplotlib/ipympl/pull/461 but stalled out bc I couldn't get the tests all passing

mwcraig replied:
All of these will need new releases for ipywidgets 8 -- the pins should make sure that no one gets an 8-only package.

One reason for setting the goal of presenting in 8 is to help work through the remaining issue.

mwcraig commented:
> Hey @mwcraig is not working at all for me. I had a problem with `couldn't find version 0.2.19 of voila` so I changed to the latest `voila==0.3.5` and had a lot of conflicts. Changed back to `0.2.10` and looks fine. We surely need the 2.19?

Thanks -- it turns out we can use 0.3.5 if we drop `voila-vuetify`, which I think is fine, so I made that change.

mwcraig commented:
> The readme claims that:
> 
> > The installation instructions were tested on an up-to-date version of Windows 10 Professional. If you encounter any issues on Windows please open an issue or contact us through slack.
> 
> Is that actually true? Does anyone have windows on which to test this?

It will be in about 30 minutes -- I have a windows 10 VM on my work laptop.
